src/Symfony/Component/Form/FormFactory.php

```diff
@@ -61,36 +61,12 @@ public function createForProperty($class, $property, $data = null, array $option
      */
     public function createBuilder($type = 'Symfony\Component\Form\Extension\Core\Type\FormType', $data = null, array $options = array())
     {
-        $name = null;
-        $typeName = null;
-
         if ($type instanceof ResolvedFormTypeInterface) {
```

Comment:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/FormFactoryInterface.php must be updated as well as it still says `string|FormTypeInterface $type`